src/middlewared/middlewared/plugins/catalogs_linux/features.py

```diff
@@ -8,6 +8,8 @@
 SUPPORTED_FEATURES = {
     'normalize/interfaceConfiguration',
     'normalize/ixVolume',
+    'definitions/certificate',
+    'definitions/certificate_authority',
```

Comment:
Why everything else is camelCase but here we have underscore?

Reply:
Thank you for catching that :) It should be camel case here as well